src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -452,10 +459,16 @@ namespace winrt::Microsoft::Terminal::Control::implementation
     {
         auto settings{ _core.Settings() };
         auto bgColor = til::color{ _core.FocusedAppearance().DefaultBackground() };
+
+        auto transparent_bg = settings.BgImageForWindow();

```

Comment:
_super nit_: stylistically we usually use `camelCase` for variable names (e.g. `transparentBg`), but I wouldn't have even bothered with this if there weren't other comments on the PR. We also don't have that codified anywhere so that's on me 😉

Reply:
There's another `transparent_bg` left in the PR. But that's a "super nit" indeed. 😅